src/libraries/System.Text.Json/src/System/Text/Json/Reader/JsonReaderHelper.NoIntrinsics.cs

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why two copies of the file, rather than an internal shim for intrinsics that always returns `false`/`throws`?

Reply:
Looks like I asked this a while back here: https://github.com/dotnet/runtime/pull/41097#discussion_r558589116 and then @steveharter asked for a file split here: https://github.com/dotnet/runtime/pull/41097#discussion_r593327402

I am really not a fan of duplicating the `Vector<T>` and fallback logic across two files just so the intrinsic logic can be cleanly separated. The JIT (even full framework) should correctly handle dead code elimination for properties that simply return `false` and it will not be pleasant remembering to update both copies of the files for half the code.

Reply:
Don't see any `*.PlatformNotSupported.cs` references in [System.Text.Encodings.Web.csproj](https://github.com/dotnet/runtime/blob/5bc2acebdca3c5fa680e2144751f9e2f437da2a0/src/libraries/System.Text.Encodings.Web/src/System.Text.Encodings.Web.csproj)?

Reply:
Looks like @GrabYourPitchforks removed them in https://github.com/dotnet/runtime/pull/49373